### PR TITLE
feat(Range): add Range component

### PR DESCRIPTION
**Pull request by youssefssc**
https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/DS-Atoms?node-id=13343%3A8984

Suggestions on how to resolve the type errors are welcome

### CODE REVIEW

**Comment by youssefssc** [comment]
type errors fixed, thanks! @ajkl2533

File: src/components/forms/Range/Range.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  title: 'components/forms/Range',
+  component: Range,
+  argTypes: {
+    min: { control: 'number' },
+    max: { control: 'number' },
+    defaultValue: { control: 'number' },
+    isProgressRight: { control: 'boolean' },
+    hasSideInput: { control: 'boolean' },
+    isDisabled: { control: 'boolean' },
+  },
```
Comment (ajkl2533) [inline src/components/forms/Range/Range.stories.tsx:17]: This is unnecessary and you can remove it

File: src/components/forms/Range/Range.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  argTypes: {
+    min: { control: 'number' },
+    max: { control: 'number' },
+    defaultValue: { control: 'number' },
+    isProgressRight: { control: 'boolean' },
+    isDisabled: { control: 'boolean' },
+  },
+} as Meta;
+
+export const playground: Story<
```
Comment (ajkl2533) [inline src/components/forms/Range/Range.stories.tsx:19]: Please add `const RangeTemplate` and use the notation we started using elsewhere https://www.figma.com/file/5FMXj5ydt1i7N6gU01EGVq/DS-Snowflakes?node-id=7030%3A17505
```jsx
const RangeTemplate: Story<
  RangeProps & React.InputHTMLAttributes<HTMLInputElement>
> = (args) => <Range {...args} aria-label="Input" />;

export const playground = RangeTemplate.bind({});
playground.parameters = {
  chromatic: { disable: true },
};
playground.args = { min: 0, max: 100 };
...

File: src/components/forms/Range/Range.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+export default {
+  title: 'components/forms/Range',
+  component: Range,
+  argTypes: {
+    min: { control: 'number' },
+    max: { control: 'number' },
+    defaultValue: { control: 'number' },
+    isProgressRight: { control: 'boolean' },
+    isDisabled: { control: 'boolean' },
```
Comment (ajkl2533) [inline src/components/forms/Range/Range.stories.tsx:15]: Please remove, Storybook is able to pick this from prop-types and TS annotations

File: src/components/forms/Range/Range.types.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+export interface RangeProps
```
Comment (ajkl2533) [inline src/components/forms/Range/Range.types.ts:1]: Could you please add short description of props like we have in https://github.com/securityscorecard/design-system/blob/alpha/src/components/Pill/Pill.types.ts#L45-L64
Storybook is able to pick it up and display in the props table

**Comment by youssefssc** [comment]
@ajkl2533 added type descriptions and `RangeTemplate`

**Review by ajkl2533 — APPROVED** [review]

**Pull request merged**